doc/users/text_props.rst

```diff
@@ -59,3 +60,97 @@ upper right.
 
 .. plot:: pyplots/text_layout.py
    :include-source:
+
+
+==============
+ Default Font
+==============
+
+The base default font is controlled by a set of rcParams
```

Comment:
In my native language there would be punctuation missing ( : or something
equivalent). I assume it is the same in English?

On Sep 11, 2016 9:30 AM, "Thomas A Caswell" <notifications@github.com>
wrote:

> In doc/users/text_props.rst
> <https://github.com/matplotlib/matplotlib/pull/7025#discussion_r78298077>:
>
> > @@ -59,3 +60,97 @@ upper right.
> >
> >  .. plot:: pyplots/text_layout.py
> >     :include-source:
> > +
> > +
> > +==============
> > + Default Font
> > +==============
> > +
> > +The base default font is controlled by a set of rcParams
>
> @NelleV <https://github.com/NelleV> I do not understand what you mean
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/7025/files/e6029e7ee98d8da19dc102f87e27f8e5b83f0d45#r78298077>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALR3toFyaoG8teHf-efKYVHm9nAakrBks5qpCyrgaJpZM4J0sWK>
> .
>